### ISSUE TITLE
Review sensor frames

### ISSUE THREAD

**Issue by PeterBowman** [roboticslab-uc3m/teo-configuration-files#32]
In the context of https://github.com/roboticslab-uc3m/yarp-devices/issues/263, I observed that sensor axes do not match the TCP frame [described in our manual](https://github.com/roboticslab-uc3m/teo-developer-manual/blob/adf427c338987bc5b11c9a321c92e44a202b49c6/docs/fig/dh-drawing.png). I believe said frame matches the orientation of force/torque raw data streamed by the legacy [Jr3Pci implementation](https://github.com/roboticslab-uc3m/yarp-devices/tree/1c466d0977503a62629e8be86292cde7b940dd38/libraries/YarpPlugins/Jr3Pci) (in `--dextrorotary` mode). However, the proper sensor axes should be obtained by rotating about the depicted X axis by 180 degrees.

In other words, given the [above drawing](https://github.com/roboticslab-uc3m/teo-developer-manual/blob/adf427c338987bc5b11c9a321c92e44a202b49c6/docs/fig/dh-drawing.png) and focusing on the arm TCPs:

- Axis X follows the normal direction "inwards" (as depicted).
- Axis Z points up.
- Axis Y would point left.

Possible solutions:

1. Keep the current TCP frame as-is, define an additional sensor frame matching the new implementation.
2. Change the TCP frame to match the new implementation, update scripts and demos.

**Comment by PeterBowman** [comment]
3. Since there is an ongoing redesign of several joints and links and therefore (I presume) the sensors will be unmounted anyway, just mount them on their opposite side.

cc @smcdiaz

**Comment by jgvictores** [comment]
@PeterBowman Is https://robots.uc3m.es/teo-developer-manual/diagrams.html#ft-sensors-ports-channels-and-coordinate-systems accurate?

**Comment by PeterBowman** [comment]
> @PeterBowman Is https://robots.uc3m.es/teo-developer-manual/diagrams.html#ft-sensors-ports-channels-and-coordinate-systems accurate?

It would seem so, indeed, at least for the wrist sensors (I haven't checked the ankles yet).
